src/ui/components/Tour/Tour.tsx

```diff
@@ -224,7 +230,13 @@ const Tour: React.FC = () => {
           <>
             {showConsoleNavigate && showBreakpointAdd && showBreakpointEdit && <TimeTravelGif />}
             {!showConsoleNavigate && showBreakpointAdd && showBreakpointEdit && <OneClickLogsGif />}
+
             {!showConsoleNavigate && !showBreakpointAdd && showBreakpointEdit && <EditLogsGif />}
+
+            {!showConsoleNavigate && showBreakpointAdd && !showBreakpointEdit && (
+              <OneClickLogsGif />
+            )}
```

Comment:
This looks broken:
```js
            {!showConsoleNavigate && showBreakpointAdd && showBreakpointEdit && <OneClickLogsGif />}

            {!showConsoleNavigate && showBreakpointAdd && !showBreakpointEdit && (
              <OneClickLogsGif />
            )}
```

Reply:
Good call, new PR -> https://github.com/replayio/devtools/pull/9118#pullrequestreview-1407152012